code/game/objects/structures/survey_probe.dm

```diff
@@ -21,6 +21,17 @@
 	/// Timer of the survey process.
 	/// If null, it is not currently surveying.
 	var/timer_id = null
+	/// The language that the report will output in.
+	var/report_language
+	/// Should this probe start deployed? Used for mapped-in probes
+	var/start_deployed = FALSE
+
+/obj/structure/survey_probe/Initialize(mapload)
+	. = ..()
+	if(start_deployed)
+		anchored = TRUE
+		density = TRUE
+		icon_state = "surveying_probe_deployed"
```

Comment:
you should update this to use the deployment proc, so if anything ever gets added to the deployment it won't have to be copy pasted in two places